scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/definition/component/parameter/editor/ParameterTypeEditorDeterminer.scala

```diff
@@ -1,65 +1,64 @@
 package pl.touk.nussknacker.engine.definition.component.parameter.editor
 
 import pl.touk.nussknacker.engine.api.definition._
-import pl.touk.nussknacker.engine.api.editor.DualEditorMode
 import pl.touk.nussknacker.engine.api.typed.typing.{SingleTypingResult, TypingResult}
 
 import java.time.temporal.ChronoUnit
 
 class ParameterTypeEditorDeterminer(val typ: TypingResult) extends ParameterEditorDeterminer {
 
-  override def determine(): Option[ParameterEditor] = {
+  override def determine(): List[ParameterEditor] = {
     Option(typ)
       .collect { case s: SingleTypingResult =>
         s.runtimeObjType
       }
       .map(_.klass)
       .collect {
         case klazz if klazz.isEnum =>
-          DualParameterEditor(
+          List(
             FixedValuesParameterEditor(
               possibleValues = klazz.getEnumConstants.toList.map(ParameterTypeEditorDeterminer.extractEnumValue(klazz))
             ),
-            DualEditorMode.SIMPLE
+            SpelParameterEditor
           )
         case klazz if classOf[java.lang.CharSequence].isAssignableFrom(klazz) =>
-          DualParameterEditor(
-            simpleEditor = StringParameterEditor,
-            // For compatibility reasons we choose raw editor - otherwise users could write expressions in simple mode
-            defaultMode = DualEditorMode.RAW
+          List(
+            SpelParameterEditor,
```

Comment:
We don't change the default to `SpelTemplateParameterEditor` first in this change?

Reply:
Done